src/components/about/About.component.tsx

```diff
@@ -0,0 +1,20 @@
+import { TitleComponent } from '../title/Title.component';
+
+// @ts-ignore
```

Comment:
Hey, @3apy6a, thanx for a good pr.
Please don't use `@ts-ignore` comment whenever you can.
It is the same as `!imortant` in CSS - only for the cases when you can't do the stuff without it.
Please try to refactor your code and remove ignore comment.

Reply:
try to done